OptimizelySDK/Optimizely.cs

```diff
@@ -1362,6 +1362,17 @@ internal void IdentifyUser(string userId)
             Dictionary<string, object> data
```

Comment:
Don't we need to assign default values for `type = null` and `data = [:]`?

Reply:
If we add "null" type default, can we remove "(typically `fullstack` from server-side SDK events)" in line 1357.

Reply:
I've made this change. (If it looks good, I'll update the csharp-testapp.)